tests/metrics/machine_learning/onednn-dockerfile/Dockerfile

```diff
@@ -0,0 +1,44 @@
+# Copyright (c) 2024 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+
+# Set up an Ubuntu image with 'phoronix-test-suite' installed
+
+FROM ubuntu:22.04
+
+LABEL DOCKERFILE_VERSION="1.0"
+
+ENV PHORONIX_TAR_URL="https://phoronix-test-suite.com/releases/"
+ENV PHORONIX_VER="phoronix-test-suite-10.8.4.tar.gz"
+ENV PHORONIX_CFG="/usr/share/phoronix-test-suite/pts-core/static/user-config-defaults.xml"
+
+ENV DEBIAN_FRONTEND noninteractive
+ENV DEBCONF_NONINTERACTIVE_SEEN true
+
+RUN apt-get update && \
+	apt-get install -y --no-install-recommends git curl build-essential autoconf && \
+	apt-get install -y --no-install-recommends php libapache2-mod-php php-bz2 php-gd php-sqlite3 php-xml && \
+	apt-get install -y --no-install-recommends cmake cmake-data mesa-utils vulkan-tools unzip apt-file wget && \
+        curl -OkL ${PHORONIX_TAR_URL}/${PHORONIX_VER} && \
+	tar zxf ${PHORONIX_VER} && cd phoronix-test-suite && ./install-sh && \
+	apt-get clean && rm -rf /var/lib/apt/lists && \
+	rm -rf /var/lib/apt/lists/* && \
```

Comment:
is this necessary? the previous step is removing the directory

Reply:
Fixed. Thank you @GabyCT